src/shared/lib/helpers/returnStatusColor/returnStatusColor.ts

```diff
@@ -0,0 +1,13 @@
+import { StatusEnum } from '../../../../entities/Task';
+
+export const returnStatusColor = (value: StatusEnum): 'success' | 'primary' | 'secondary' => {
+    let color: 'success' | 'primary' | 'secondary';
+
+    color = value === StatusEnum.COMPLETED
```

Comment:
think it's better to use switch statement here

Reply:
Fixed